src/doc/trpl/compound-data-types.md

```diff
@@ -47,7 +47,8 @@ This pattern is very powerful, and we'll see it repeated more later.
 
 There are also a few things you can do with a tuple as a whole, without
 destructuring. You can assign one tuple into another, if they have the same
-arity and contained types.
+contained types and arity. Tuples have the same arity when they have the same
```

Comment:
Why introduce the concept of "arity" here at all? "length" should be good enough, and I've never seen the word "arity" being used in this context in discussions about Rust.